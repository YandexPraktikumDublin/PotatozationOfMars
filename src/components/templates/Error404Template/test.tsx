import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Error404Template } from '.'

describe('<Error404Template />', () => {
  it('should renders correct <Error404Template />', () => {
    const wrapper = shallow(<Error404Template />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
