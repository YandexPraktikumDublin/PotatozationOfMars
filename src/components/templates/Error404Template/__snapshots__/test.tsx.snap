// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error404Template /> should renders correct <Error404Template /> 1`] = `<Fragment />`;
